tools/tesseratos/include/tesseratos/console/plugin.hpp

```diff
@@ -0,0 +1,26 @@
+/// @dir
+/// @brief @ref tesseratos-console-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup tesseratos-console-plugin
+
+#pragma once
+
+#include <cubos/engine/assets/plugin.hpp>
+
+namespace tesseratos
+{
+    /// @defgroup tesseratos-console-plugin Asset explorer
+    /// @ingroup tesseratos
+    /// @brief Displays Cubos Logs.
+    ///
+    /// ## Dependencies
+    /// - @ref imgui-plugin
+    /// - @ref log-plugin
```

Comment:
This is not a plugin, right?

Reply:
Todos os outros têm commented isso so im not sure mas diria que sim